source/about/faq-license.rst

```diff
@@ -122,7 +122,7 @@ As we add enterprise-only functionality, we will update the license on affected
 To which repositories does this apply?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-We plan to apply this license to the enterprise directories of our `Jira <https://github.com/mattermost/mattermost-plugin-jira>`_, `Microsoft Calendar <https://github.com/mattermost/mattermost-plugin-mscalendar>`_, and `Microsoft Teams Meetings <https://github.com/mattermost/mattermost-plugin-msteams-meetings>`_ plugins. We also intend to release :doc:`Playbooks </guides/repeatable-processes>` and `Channel Export <https://github.com/mattermost/mattermost-plugin-channel-export>`_ plugins under the Mattermost Source Available License. New, Mattermost-authored plugins will generally be released under the Mattermost Source Available License. When we update the licenses, we will release a new version and note the change in the ``README.md`` file of the GitHub repository and any release notes.
+The Mattermost :doc:`Playbooks </guides/repeatable-processes>` and `Channel Export <https://github.com/mattermost/mattermost-plugin-channel-export>`_ plugins are available under the Mattermost Source Available License as well as the enterprise directories of our `Jira <https://github.com/mattermost/mattermost-plugin-jira>`_, `Microsoft Calendar <https://github.com/mattermost/mattermost-plugin-mscalendar>`_, and `Microsoft Teams Meetings <https://github.com/mattermost/mattermost-plugin-msteams-meetings>`_ plugins.  New, Mattermost-authored plugins will generally be released under the Mattermost Source Available License. When we update the licenses, we will release a new version and note the change in the ``README.md`` file of the GitHub repository and any release notes.
```

Comment:
@lieut-data Can you please confirm the Calendar and MS Teams Meeting plugins, I could not tell if they were actually converted and with move to community supported, this change may have been reverted.

Reply:
I agree. It seems too much to manage in docs.